course_discovery/apps/api/v1/views/course_runs.py

```diff
@@ -163,15 +170,23 @@ def contains(self, request):
               type: string
               paramType: query
               multiple: false
+            - name: org
+              description: Filter results on edx organization's short name.
+              required: false
+              type: string
+              paramType: query
+              multiple: false
         """
         query = request.GET.get('query')
         course_run_ids = request.GET.get('course_run_ids')
         partner = self.request.site.partner
+        edx_org_short_name = request.GET.get('org')
 
         if query and course_run_ids:
             course_run_ids = course_run_ids.split(',')
-            course_runs = CourseRun.search(query).filter(partner=partner.short_code).filter(key__in=course_run_ids). \
```

Comment:
The previous code should not be altered. We need to minimize our changes.

Reply:
Done

Reply:
ok